app/views/topics/index.html.erb

```diff
@@ -14,4 +14,6 @@
     <% end %>
   </table>
 
+  <%= link_to 'Add your own topics here', new_topic_path %>
```

Comment:
How 'bout a shorter link name. The convention is to use the words Rails uses, so 'New topic'.